### DISCUSSION TITLE
Welcome to aws-eda-slurm-cluster Discussions!

### DISCUSSION THREAD

**Discussion by cartalla** [aws-samples/aws-eda-slurm-cluster#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * If you have a question you can ask here before opening an issue.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lihnm** [comment]
I'm having issue when bootstrapping cdk. It successfully bootstrapped when I run this: 

> cdk bootstrap aws://xxxxxxx/us-east-1 -c region=us-east-1 -c account_id=xxxxxx -c stack_name=slurmminimal -c SshKeyPair=key -c VpcId=vpc-0ff9de2afc42ab3 -c SubnetId=subnet-0456bccaac4a63914 -c ErrorSnsTopicArn=arn:aws:sns:us-east-1:098962443:ErrorSlurm -c SubmitterSecurityGroupIds=eyJzZ1NsdXJtIjogInNnLTA0MjgxZDc2ZjljNGEwZGNmIn0=

But when I minimal deploy with 

> ./install.sh --prompt --cdk-cmd create

CDK would bootstrap again and failed to get a value from App()

> ERROR:root:No token file found. Also make sure that a [prod] section with a 'token = value' assignment exists.
> Traceback (most recent call last):
> File "app.py", line 24, in <module>
> app = App()

I have configured aws secret id and key, and also region. The user and ec2 roles don't have any IAM permission.

**Reply by cartalla** [reply]
You need a lot of IAM permissions to deploy the cluster so that is likely an issue. This is documented here:

https://aws-samples.github.io/aws-eda-slurm-cluster/deploy/

I don't have a more restrictive list of required IAM permissions than Admin right now.

**Reply by lihnm** [reply]
I'm sorry that I did not elaborate 

The user and ec2 don't have IAMFullAccess permission. But they do have full access permission for Cloudformation, SSM, Ec2, S3, Lambda, SNS, Cloudwatch and VPC. The reason I did not give IAMFullAccess is becasue cdk would me the error: 

> Need to perform AWS calls for account unknown-account, but no credentials found. Tried: default credentials.

What permission do you think I am missing here ?

**Reply by lihnm** [reply]
I've tried AdministratorAccess already but CDK bootstrapping still cannot find token value as above

**Reply by cartalla** [reply]
I'm a little bit confused. The last message says no credentials found. How are you configuring your credentials?
To get started you should configure credentials that give administrative permissions.

**Reply by cartalla** [reply]
I created a new issue. Let's continue there. Issue #31
